sycl/source/queue.cpp

```diff
@@ -13,6 +13,19 @@
 
 __SYCL_INLINE namespace cl {
 namespace sycl {
+
+namespace detail {
+
+QueueOrder getQueueOrder(const property_list &propList) {
+  if (propList.has_property<property::queue::in_order>()) {
+    return QueueOrder::Ordered;
+  } else {
+    return QueueOrder::OOO;
+  }
```

Comment:
Please, do not use `else` after `return`.
```suggestion
  if (propList.has_property<property::queue::in_order>()) {
    return QueueOrder::Ordered;
  }
  return QueueOrder::OOO;

Reply:
Done.